test/unit/roving-tabindex.test.ts

```diff
@@ -0,0 +1,166 @@
+import { LitElement, customElement } from 'lit-element';
+import { expect, fixture, html } from '@vaadin/vaadin-component-dev-dependencies/testing.js';
+import {
+  arrowDown,
+  arrowLeft,
+  arrowRight,
+  arrowUp,
+  end,
+  home,
+  tab
+} from '@vaadin/vaadin-component-dev-dependencies/keys.js';
+import { KeyboardDirectionMixin } from '../../src/keyboard-direction-mixin';
+import { RovingTabindexMixin } from '../../src/roving-tabindex-mixin';
+import { SlottedItemsMixin } from '../../src/slotted-items-mixin';
+
+const { sinon } = window;
+
+@customElement('rtm-element')
+class RtmElement extends RovingTabindexMixin(KeyboardDirectionMixin(SlottedItemsMixin(LitElement))) {
+  render() {
+    return html`
+      <style>
+        :host {
+          display: block;
+        }
+
+        div {
+          display: flex;
+          flex-direction: column;
+        }
+      </style>
+      <div>
+        <slot></slot>
+      </div>
+    `;
+  }
+}
+
+describe('RovingTabindexMixin', () => {
+  let element: RtmElement;
+  let items: HTMLElement[];
+
+  const expectTabindex = (order: number[]) => {
+    order.forEach((val, idx) => expect(items[idx].tabIndex).to.be.equal(val));
+  };
```

Comment:
I would rename this method to `expectTabIndex` (with capital "i") or maybe even `expectTabIndexes` (since it's checking more than one). (Grammar note: apparently the plural of "index" can be "indexes" or "indices" but in this case "indexes" is probably easier to understand since it's referring to "tabIndex")

Also using this method could be simplified by only requiring to give a single number as the parameter which would define which item is the one that should have `tabIndex` of `0`. It should still check that all others have `tabIndex` of `1` but then all calls to this method in the tests would be much simpler. Maybe the method name could also be changed to something that is easier to understand if this is done.

I don't think there are any cases where currently where it is not expected to have exactly one item with `tabIndex` of `0` and all the rest as `-1`. Even if it was, there should still be some shorthand to expect that a specific index should have `tabIndex` of `0` (this could be a new method instead of changing the current one).

Not a blocker, but I think it would nicely simplify the expect calls in the tests.

Reply:
Thanks, done.